core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -164,6 +165,13 @@ private void reset() {
         loadTimeout.cancel();
         heart.stopBeatingHeart();
         exoPlayer.release();
+        destroySurface();
+    }
+
+    private void destroySurface() {
+        if (playerView != null) {
+            playerView.getContainerView().setVisibility(View.GONE);
```

Comment:
I'm worried that others will be confused by this. `destroySurfaceByRemoving` or something similar...